applications/nrf5340_audio/src/bluetooth/bt_stream/unicast/unicast_client.c

```diff
@@ -749,9 +1034,15 @@ static void work_stream_start(struct k_work *work)
 		work_data.retries);
 
 	if (work_data.dir == BT_AUDIO_DIR_SINK) {
```

Comment:
Can we discuss the naming of SINK and SOURCE related to which device we are on?

Reply:
Sure